mods/sfinv/api.lua

```diff
@@ -57,6 +56,10 @@ function sfinv.make_formspec(player, context, content, show_inv, size)
 	return table.concat(tmp, "")
 end
 
+function sfinv.get_homepage_name(player)
+	return sfinv.homepage_name or "sfinv:crafting"
```

Comment:
No need to support the sfinv which has been in use for a few days in a dev version.